src/runtime/mod.rs

```diff
@@ -92,6 +95,10 @@ pub struct Module {
     ///
     /// If none is supplied, then http://localhost:8080/v1 is used
     pub bindle_server: Option<String>,
+
+    /// List of hosts that the guest module is allowed to make HTTP requests to.
+    /// If none is supplied, the guest module can send requests to any server.
+    pub allowed_hosts: Option<Vec<String>>,
```

Comment:
The other PR from the http modules says that if `None` is specified, it can't make requests to any server, but it says the opposite here

Reply:
Ah, good catch.
Changing now, thanks!